src/symbol.rs

```diff
@@ -24,7 +24,7 @@ mod sym;
 
 pub use frame::{Frame, FrameIter, Local, LocalValue};
 pub use relocate::RelocatedDwarf;
-pub use source::DisassemblySource;
+pub use source::{pretty, CrabResult, DisassemblySource, Snippet};
```

Comment:
```suggestion
pub use source::{CrabResult, DisassemblySource, Snippet};
```

No need to re-export `pretty`.

Reply:
I see.